docs/public-networks/how-to/use-pow/mining.md

```diff
@@ -7,7 +7,13 @@ tags:
   - private networks
 ---
 
-# Configure mining
+# Configure mining (Deprecated)
```

Comment:
may need some more nuance added since mining is still used in QBFT. @siladu can you clarify? would this include miner-start and stop APIs

Reply:
resolved per besu PR